Cargo.toml

```diff
@@ -14,13 +14,14 @@ rand = "0.8.5"
 ratatui = { version = "0.29.0", optional = true }
 rusqlite = { version = "0.30.0", features = ["bundled"] }
 shellexpand = "3.1.0"
+simple-macros = { path = "simple-macros" }
 tabled = "0.17.0"
 yaml-rust = "0.4"
 
 [features]
-default = ["log_info", "log_warn", "log_error"]
-log_info = []
-log_debug = []
-log_warn = []
+default = ["log_info"]
```

Comment:
Is this configured the way you want? You don't want "log_info = ['log_info']"?

Reply:
Yeah these are basically feature flags and not really related to the macros themselves.

In the "log_info" macro, we check if the "log_info" feature flag is set and if not, we dont run the code.